### ISSUE TITLE
Rename it no longer working on Sketch and macOS Ventura.

### ISSUE THREAD

**Issue by edwardsanchez** [rodi01/RenameIt#151]
Whenever I select the rename options, nothing happens.

**Comment by Polyphian** [comment]
> Whenever I select the rename options, nothing happens.

Yeah, same. There's another renaming plugin I've tried using and that doesn't work either. I relied on this plugin too much apparently because it's absolutely destroying my workflow by not being able to use it.

This plugin is no longer maintained and hasn't been updated in years. If you find an alternative please let me know.

**Comment by PietDAmore** [comment]
Same here, thanks looks like this is known for quite some time now? Can we do something to fix this?

**Comment by ciscoCid** [comment]
Well the best way would be to pay the owner. Worth starting a crowdfund.

**Comment by ciscoCid** [comment]
Or just pay him pizza 😆

**Comment by rodi01** [comment]
Unfortunately, the issue is with the plugin's architecture and the outdated tools that were used during its development. In order to resolve the problem, a substantial rewrite of the plugin would be necessary. However, since the plugin is no longer being actively developed, addressing the issue may not be possible.
